lib/rules/no-console.js

```diff
@@ -36,30 +113,60 @@ module.exports = {
     },
 
     create: function(context) {
+        var globalScope;
 
         return {
+            "Program": function() {
+                globalScope = context.getScope();
+            },
 
             "MemberExpression": function(node) {
+                var currentScope = context.getScope(),
+                    passedProperty = getPropertyName(node),
+                    nestedNode = node.object,
```

Comment:
would something like 'passedObject' be a better name? It would be more consistent with using 'passedProperties'.